clippy.toml

```diff
@@ -1,13 +1,10 @@
 doc-valid-idents = [
+  "GameActivity",
   "GilRs",
   "glTF",
-  "macOS",
   "NVidia",
   "OpenXR",
-  "sRGB",
   "VSync",
-  "WebGL2",
-  "WebGPU",
```

Comment:
While I'm here, I removed several idents, due to them already being in the default list of identifiers. https://rust-lang.github.io/rust-clippy/rust-1.88.0/index.html#doc_markdown